Userland/Libraries/LibWeb/HTML/HTMLButtonElement.cpp

```diff
@@ -57,18 +57,18 @@ void HTMLButtonElement::initialize(JS::Realm& realm)
     set_prototype(&Bindings::ensure_web_prototype<Bindings::HTMLButtonElementPrototype>(realm, "HTMLButtonElement"));
 }
 
-DeprecatedString HTMLButtonElement::type() const
+StringView HTMLButtonElement::type() const
 {
     auto value = deprecated_attribute(HTML::AttributeNames::type);
 
 #define __ENUMERATE_HTML_BUTTON_TYPE_ATTRIBUTE(keyword, _) \
     if (value.equals_ignoring_ascii_case(#keyword##sv))    \
-        return #keyword;
+        return #keyword ""sv;
```

Comment:
Sorry, missed this before - the standard way to do this is:
```c++
return #keyword##sv;
```

The `equals_ignoring_ascii_case` invocation above this shows how it works

Reply:
Ah, good ol' preprocessor magic - thanks :) fixed